metacatalog/ext/standard_export/__init__.py

```diff
@@ -0,0 +1 @@
+from .extension import StandardsExportExtension
```

Comment:
The extension should be called `standards_export` as it is not the standard export extension. Also to be in line with the Class name

Reply:
But as this is a Class, we should use camel case and no separation between words, right?

Reply:
Right, but the class has already the right name. Just the File doesn't have the right name.